plotnine/scales/scale_color.py

```diff
@@ -357,7 +357,10 @@ def __init__(self, type='seq', palette=1, values=None, **kwargs):
         # Grab 7 colors from brewer and create a gradient palette
         # An odd number matches the midpoint of the palette to that
         # of the data
-        colors = brewer_pal(type, palette)(7)
+        # See https://github.com/has2k1/plotnine/issues/377
+        # TODO: make direction parameter -1 by default: 
+        # https://github.com/tidyverse/ggplot2/blob/master/R/scale-brewer.r#L89
+        colors = reversed(brewer_pal(type, palette)(7))
```

Comment:
You can add the `direction` parameter to scale colour brewer. Support for it was added in [mizani v0.6.0](https://mizani.readthedocs.io/en/stable/palettes.html#mizani.palettes.brewer_pal), I just forgot to add it to plotnine right after releasing mizani.

Reply:
Done. How do you do the dependencies? Is it ok to add mizani=0.6.0 dependency?

Reply:
Oh mizani >= 0.6.0 is already a requirement, sweet.